waspc/data/Generator/templates/react-app/src/storage.ts

```diff
@@ -1,4 +1,5 @@
 export type DataStore = {
+    prefixedKey(key:string): string;
```

Comment:
```suggestion
    getPrefixedKey(key: string): string;
```

Since it's a function let's use a verb